docs/get-started/deployment-and-usage.md

```diff
@@ -99,6 +99,51 @@ this approach does not guarantee running once on every node. For example,
 tainted, non-ready nodes or some other reasons in Job scheduling may cause some
 node(s) will run extra job instance(s) to satisfy the request.
 
+### Helm Chart
```

Comment:
Similarly to Deployment Templates heading above , perhaps : Deployment with Helm